lib/fluent/plugin/in_tail.rb

```diff
@@ -1095,7 +1095,7 @@ def handle_notify
                       break
                     end
                   end
-                rescue EOFError
+                rescue EOFError, Errno::ESTALE
```

Comment:
Isn't it needed to output some warning or info log message for this?

Reply:
> Isn't it needed to output some warning or info log message for this?

Pushed a new commit 331ac72 to incorporate that feedback. Added a
sepearate `reque` and called `log.info` within that block.